page-recent-news.php

```diff
@@ -0,0 +1,212 @@
+<div class="layout-container full--until-large">
+  <div class="flex-container cf">
+    <div class="shift-left--fluid column__primary bg--white no-pad--top no-pad--btm can-be--dark-light">
+      <?php $carousel_format = get_field('carousel_type');?>
+      <?php if ($carousel_format == "small_format_inset"): ?>
+        <?php include(locate_template('patterns/components/hero-carousel.php')); ?>
+      <?php endif; ?>
+      <div class="g g-2up--at-medium with-divider">
+        <div class="gi">
+          <div class="spacing">
+            <div class="spacing--half">
+              <h2 class="font--tertiary--l theme--primary-text-color pad pad-double--top pad-half--btm">
+                Recent news
+              </h2>
+            </div>
+            <hr>
+            <?php
+              // Recent News
+              $news = array(
+                'cat' => array(14),
+                'category__not_in' => array(19,20),
+                'posts_per_page' => 4,
+                'tax_query' => array( array(
+                  'taxonomy' => 'post_format',
+                  'field' => 'slug',
+                  'terms' => array('post-format-video' , 'post-format-gallery'),
+                  'operator' => 'NOT IN'
+                ))
+              );
+              query_posts($news);
+            ?>
+            <?php while (have_posts()) : the_post(); ?>
+              <div class="pad">
+                <?php
+                  $title = get_the_title();
+                  $intro = get_field('intro');
+                  $body = strip_tags(get_the_content());
+                  $excerpt_length = 100;
+                  $image = get_post_thumbnail_id();
+                  $button_text = 'Read More';
+                  $date = get_the_date();
+                  $button_url = get_the_permalink();
+                  $round_image = get_sub_field('make_the_image_round');
+                  $thumbnail = wp_get_attachment_image_src($image, "horiz__4x3--s")[0];
+                  $thumbnail_round = wp_get_attachment_image_src($image, "square--s")[0];
+                  $alt = get_post_meta($image, '_wp_attachment_image_alt', true);
+                  $block_inner_class = 'block__row--small-to-large';
+                  if (isset($post->post_date) && $post->post_type == 'post') {
+                    $date = get_the_date('M j, Y');
+                    $date_formatted = get_the_date('c');
+                  }
+                ?>
+                <?php include(locate_template('patterns/blocks/block-media.php')); ?>
+              </div>
+              <hr>
+            <?php endwhile; ?>
+            <?php wp_reset_query(); ?>
+          </div><!-- ./spacing -->
+        </div><!-- ./gi -->
+        <div class="gi pad-double--btm">
+          <div class="spacing">
+            <div class="spacing--half">
+              <h2 class="font--tertiary--l theme--primary-text-color pad pad-double--top pad-half--btm">
+                ANN Video
```

Comment:
I'm thinking these might have to be dynamic. I'm not sure how they'll get filled yet, but I'm fairly certain they won't be hard-coded.